cypress/integration/instruments.ts

```diff
@@ -68,7 +68,7 @@ context('Instrument tests', () => {
     });
 
     it('User officer should be able to create instrument', function () {
-      if (featureFlags.getEnabledFeatures().get(FeatureId.EXTERNAL_AUTH)) {
+      if (!featureFlags.getEnabledFeatures().get(FeatureId.SCHEDULER)) {
```

Comment:
I agree, but enabling these tests would be a different PR. I have a suspicion that they have been disabled because they were failing for some other reason